templates/main.template

```diff
@@ -100,165 +126,239 @@ Parameters:
     AllowedPattern: ^[0-9a-zA-Z]+([0-9a-zA-Z-.]*[0-9a-zA-Z])*$
     ConstraintDescription: Quick Start bucket name can include numbers, lowercase
       letters, uppercase letters, periods (.), and hyphens (-). It cannot start or
-      end with a hyphen (-).
+      end with a hyphen (-). If you are unsure, do not change this value.
```

Comment:
I'm fine to include this, but these will eventually get overwritten from our upstream schema (WIP)

Reply:
Ok, if you have a better way of telling customers to not touch this parameter, let me know. I am not sold on this idea, just a thought.